Task_02/EvenChar.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Text;
+
+
+namespace Task_02
+{
+    /// <summary>
+    /// This class contains method which find substring with even characters
+    /// </summary>
+    class EvenChar
```

Comment:
Bad unclear class name, reading the name I suppose that this is even chars, but not sequence or class to process something